c/include/ml-api-service.h

```diff
@@ -115,6 +115,19 @@ int ml_service_get_pipeline (const char *name, char **pipeline_desc);
  */
 int ml_service_delete_pipeline (const char *name);
 
+/**
+ * @brief handle for ml_service. To support multiple instance for a single service, user should mangage pipeline with this handle.
+ */
+typedef void *ml_service_h;
+
+/** @todo Add description and decide final API name */
+int ml_service_launch_pipeline (const char *name, ml_service_h *handle);
+int ml_service_start_pipeline (ml_service_h handle);
+int ml_service_stop_pipeline (ml_service_h handle);
+int ml_service_destroy_pipeline (ml_service_h handle);
+int ml_service_getstate_pipeline (ml_service_h handle, ml_pipeline_state_e *state);
+int ml_service_getdesc_pipeline (ml_service_h handle, char **desc);
```

Comment:
Let's discuss function name prefix. Also we should update API description before starting the ACR process later.